Applications/Games/Origin/Local (Legacy)/script.js

```diff
@@ -1,6 +1,6 @@
 const LocalInstallerScript = include("engines.wine.quick_script.local_installer_script");
-const Downloader = include("utils.functions.net.download");
-const { Extractor } = include("utils.functions.filesystem.extract");
+const Resource = include("utils.functions.net.resource");
+const {Extractor} = include("utils.functions.filesystem.extract");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4764979)